qa/backwards/shared/src/test/java/org/elasticsearch/test/ESBackcompatTestCase.java

```diff
@@ -189,7 +189,7 @@ protected TestCluster buildTestCluster(Scope scope, long seed) throws IOExceptio
         TestCluster cluster = super.buildTestCluster(scope, seed);
         logger.info("Build internal cluster at {}", new Object[] {cluster.httpAddresses()});
         ExternalNodeServiceClient nodeServiceClient = new ExternalNodeServiceClient();
-        ExternalNode externalNode = new ExternalNode(backwardsCompatibilityVersion(), randomLong(), nodeServiceClient, new NodeConfigurationSource() {
+        ExternalNode externalNode = new ExternalNode(backwardsCompatibilityVersion().toString(), randomLong(), nodeServiceClient, new NodeConfigurationSource() {
```

Comment:
I kind of like keeping it a string because on the ExternalNodeService level "version" is just "suffix of the directory name into which elasticsearch was un-tar-ed." This seems like a reasonable spot to switch from Version to String.